tests/base_scraper/test_base_scraper.py

```diff
@@ -74,5 +74,39 @@ def test_notimplementedexceptions(self):
         self.assertRaises(mozdownload.NotImplementedError,
                           scraper.build_filename, 'invalid binary')
 
+    def test_authentication(self):
+        """testing with basic authentication"""
+        username = 'mozilla'
+        password = 'mozilla'
+        basic_auth_url = 'http://mozqa.com/data/mozqa.com/http_auth/basic/'
+
+        # test with invalid authentication
+        scraper = mozdownload.DirectScraper(directory=self.temp_dir,
+                                            url = basic_auth_url,
+                                            version=None)
```

Comment:
I was wrong here. Sorry. We should indeed include the log level and set it to ERROR. It's important for our test runs. Thanks.